### PR TITLE
Update jenkins.sysconfig.in to mention /var/cache/jenkins

### PR DESCRIPTION
**Pull request by javabrett**
Updated jenkins.sysconfig.in to mention /var/cache/jenkins to add the remaining gotcha directory if running from `jenkins.war` and having run already, changed `JENKINS_USER`

### CODE REVIEW

**Comment by javabrett** [comment]
Also ran-into [JENKINS-12875](https://issues.jenkins-ci.org/browse/JENKINS-12875), [JENKINS-36153](https://issues.jenkins-ci.org/browse/JENKINS-36153) and had to logout, clear cookies.

File: suse/build/SOURCES/jenkins.sysconfig.in [hunk 1]
```diff
@@ -38,7 +38,11 @@ JENKINS_JAVA_HOME=""
 #
 # Unix user account that runs the Jenkins daemon
 # Be careful when you change this, as you need to update
-# permissions of $JENKINS_HOME and /var/log/@@ARTIFACTNAME@@.
+# permissions of $JENKINS_HOME and /var/log/@@ARTIFACTNAME@@,
+# and if you have already run Jenkins, potentially other
+# directories such as /var/cache/@@ARTIFACTNAME@@ .
+# Existing users may also need to clear-cookies and login
```
Comment (daniel-beck) [inline suse/build/SOURCES/jenkins.sysconfig.in]: No hyphen

**Comment by daniel-beck** [comment]
@javabrett More information about the cookie issue would be helpful. The only bad cookie situation I am aware of is too many restarts, which creates different cookies every time, eventually it becomes too long. Unrelated to config changes.

**Comment by javabrett** [comment]
@daniel-beck more than happy to drop the second commit and not mention cookies, and will do that.  I certainly experienced the symptoms in the above issues - whether it was due to restarts or `JENKINS_USER` user change I don't know.  I certainly had the symptoms of JENKINS-36153 - could have been caused by some bad permissions during restart.  Also the Jenkins user lacked a proper home dir before the change, but had one after.

**Comment by javabrett** [comment]
Any further feedback on this?  I pulled the mention of the cookies issue. So should be good to merge.

**Pull request merged**